app/javascript/javascripts/debug_files/index.js

```diff
@@ -1,10 +1,11 @@
 $(document).on('turbolinks:load', function () {
-  $('.debug-file-toggle').click(function () {
+  $('.debug-file-toggle').on('click', function () {
     debug_file_id = $(this).data('id');
     $('#debug-file-metadata-' + debug_file_id).toggleClass('d-none');
+    alert('dddd');
   });
 
-  $('.destroy-debug-file').click(function () {
+  $('.destroy-debug-file').on('click', function () {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/tryzealot/zealot/pullRequest?prid=7817188)
```suggestion
  $(".destroy-debug-file").on('click', function () {
```